controllers/about/index.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const express = require('express');
+
+const { basicContent, flexibleContent } = require('../common');
+
+const router = express.Router();
+
+router.get('/', flexibleContent());
+// @TODO: Can we move the our people section to About structure? Add "profile" type to flexible content
```

Comment:
We already have a thorny page at https://www.tnlcommunityfund.org.uk/about/our-people/advisory-group-membership which is a good argument to migrate these pages to flexible content with a new "profile" content type.

Reply:
Could be a task for the IT team?